Dockerfile

```diff
@@ -7,7 +7,7 @@ MAINTAINER Dan Liew <daniel.liew@imperial.ac.uk>
 
 ENV LLVM_VERSION=3.4 \
     SOLVERS=STP:Z3 \
-    STP_VERSION=master \
+    STP_VERSION=2.1.0 \
```

Comment:
@andreamattavelli Our TravisCI builds don't currently use that version...
That should be fixed first so we actually know it works.